utils/jest-global-setup.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * MIT License
+ *
+ * Copyright (c) 2020-present, Elastic NV
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ */
+
+import { spawn } from 'child_process';
+import { wsEndpoint } from './test-config';
+
+module.exports = async () => {
+  if (wsEndpoint) {
+    return new Promise((resolve, reject) => {
+      console.log(`\nRunning BrowserService ${wsEndpoint}`);
+      const process = spawn('node', ['./dist/browser-service.js']);
+      (global as any).__browserServiceProcess = process;
+      process.stdout.on('data', data => {
+        if (data.indexOf('Listening on port: 9322') >= 0) {
```

Comment:
Instead of doing this, We should get rid of the console logs in here https://github.com/elastic/synthetics/blob/b55ab0a3812cdb301f3eef58e85ebf4718c56ccf/src/browser-service.ts#L39-L46

and move them as debug logs. We don't want to show those logs in production anyway.

Reply:
I think we can add a debug mode for these logs, I've added an item to the original issue for this.
We still need a way to confirm that browserService is up otherwise tests will be flaky.

Reply:
We already have a listening log here which should tell if its up https://github.com/elastic/synthetics/blob/b55ab0a3812cdb301f3eef58e85ebf4718c56ccf/src/browser-service.ts#L51

For other places, so you can just use our internal logger. https://github.com/elastic/synthetics/blob/b55ab0a3812cdb301f3eef58e85ebf4718c56ccf/src/core/logger.ts#L40